bot/utils/pavlov.py

```diff
@@ -109,21 +109,36 @@ async def check_perm_captain(ctx, server_name: str = None, global_check: bool =
     return True
 
 
-async def exec_server_command(ctx, server_name: str, command: str):
+async def exec_server_command(ctx, server_name: str, command: str, polling: str = "False"):
     pavlov = None
-    if hasattr(ctx, "pavlov"):
-        pavlov = ctx.pavlov.get(server_name)
-    if not hasattr(ctx, "pavlov") or pavlov is None:
-        server = servers.get(server_name)
-        pavlov = PavlovRCON(
-            server.get("ip"),
-            server.get("port"),
-            server.get("password"),
-            timeout=RCON_TIMEOUT,
-        )
-        if not hasattr(ctx, "pavlov"):
-            ctx.pavlov = {server_name: pavlov}
-        else:
-            ctx.pavlov[server_name] = pavlov
-    data = await pavlov.send(command)
-    return data
+    if polling == "False":
+        if hasattr(ctx, "pavlov"):
+            pavlov = ctx.pavlov.get(server_name)
+        if not hasattr(ctx, "pavlov") or pavlov is None:
+            server = servers.get(server_name)
+            pavlov = PavlovRCON(
+                server.get("ip"),
+                server.get("port"),
+                server.get("password"),
+                timeout=RCON_TIMEOUT,
+            )
+            if not hasattr(ctx, "pavlov"):
+                ctx.pavlov = {server_name: pavlov}
+            else:
+                ctx.pavlov[server_name] = pavlov
+        data = await pavlov.send(command)
+        return data
+    else:
+        if ctx != "noctx":
+            pavlov = ctx
+        elif ctx == "noctx":
```

Comment:
Instead of passing a string with `noctx` here please just pass `None` and check for `if ctx is None:` - that is way cleaner.